cmd/minishift/cmd/start.go

```diff
@@ -156,14 +164,6 @@ func runStart(cmd *cobra.Command, args []string) {
 		atexit.Exit(1)
 	}
 
-	// Set Proxy to as Shell Env
```

Comment:
IMO we should not remove it and if we remove it, we should remove it cleanly, see _shellProxyEnv_